zds/searchv2/management/commands/es_manager.py

```diff
@@ -12,17 +12,17 @@ class Command(BaseCommand):
     models = get_django_indexable_objects()
 
     def __init__(self, *args, **kwargs):
-        """Overridden because FakeChapter needs to be present for mapping.
-        Also, its mapping needs to be defined before the one of PublishedContent for parenting reasons (!!!).
+        """Overridden because FakeChapter needs to be present for schema.
+        Also, its schema needs to be defined before the one of PublishedContent for parenting reasons (!!!).
         """
 
         super().__init__(*args, **kwargs)
         self.models.insert(0, FakeChapter)
 
-        self.index_manager = ESIndexManager(**settings.ES_SEARCH_INDEX)
+        self.index_manager = SearchIndexManager(**settings.SEARCH_INDEX)
 
-        if not self.index_manager.connected_to_es:
-            raise Exception("Unable to connect to Elasticsearch, aborting.")
+        if not self.index_manager.connected_to_search:
+            raise Exception("Unable to connect to Typesense, aborting.")
```

Comment:
```suggestion
            raise Exception("Unable to connect to the search engine, aborting.")
```
Tentons d'abstraire les choses :)